sdk/search/azure-search-documents/swagger/README.md

```diff
@@ -111,7 +111,7 @@ This swagger is ready for C# and Java.
 output-folder: ../
 java: true
 use: '@autorest/java@4.1.2'
```

Comment:
Does the version of Autorest need to be updated here?

Reply:
not sure, if @srnagar has released the Autorest version with sync stack support.
if its available, we can update it.